qaframework-bdd-tests/src/test/resources/features/refapp-2.x/provider/provider.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Provider Management
+
+  Background: 
+    Given User clicks on the administration app
+    Then System loads administration page
+    And User clicks on the manage Provider link on the administration page
+    Then System loads manage provider page
+
+  @selenium
+  @provider
+  Scenario: Adding Provider
+    When User clicks on the add provider link
+    Then System loads add provider page
+    And User fills the provider form
+    And User searches for the created provider
+    Then User deletes provider forever
+
+  @selenium
+  @provider
+  Scenario: Editing Provider
+    When User searches for the created provider
+    Then User edits provider details
+
+  @selenium
+  @provider
+  Scenario: Retiring Provider
+    When User searches for the created provider
+    Then User retires a provider
```

Comment:
It would be a good idea to capture all the steps in a user story. For example in the above Scenario you seem to have combined more than one step into one which is not the ideal for BDD.